src/xcode/ENA/ENA/Source/Extensions/PDFDocument+EmbedImage.swift

```diff
@@ -0,0 +1,59 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import PDFKit
+
+extension PDFDocument {
+	
+	/// Embeds an Image on the first Page on given position
+	/// Inspired  by https://pspdfkit.com/blog/2019/insert-image-into-pdf-with-swift/
+	func embed(image: UIImage, at position: CGPoint) {
```

Comment:
I wouldn't know how, embed an PDF that has the "right" content and then compare the size? Create a Screenshot of booth PDFs and compare the screenshots? :/

Reply:
Or do you mean just a Test that makes sure that there is something returned?

Reply:
Maybe try to compare the data to reference data. If this is not possible or to time-consuming, at least a test which tests if the data was changed after embedding.

Reply:
Yeah I would have loved to do this. But since just creating an in memory copy without changing anything creates different data, this test would always pass, so we would not win anything. :/
<img width="554" alt="Screenshot 2021-03-09 at 17 17 54" src="https://user-images.githubusercontent.com/10122052/110606454-7f87ac80-818a-11eb-85ce-cc75c2665095.png">